params/protocol_params.go

```diff
@@ -100,6 +100,8 @@ const (
 	GetEpochSizeGas             uint64 = 1000  // Cost of querying the number of blocks in an epoch.
 	GetBlockNumberFromHeaderGas uint64 = 10000 // Cost of decoding a block header.
 	HashHeaderGas               uint64 = 20000 // Cost of hashing a block header.
+	GetParentSealBitmapGas      uint64 = 5000  // Cost of reading the parent seal bitmap from the chain.
```

Comment:
How were these values decided upon?

Reply:
`GetParentSealBitmapGas` I had very little intuition for but I've now set to get equal to the gas got the `BLOCKHASH` opcode * 12 (because it may retrieve 12 times as many blocks)
`GetVerifiedSealBitmapGas` I set to `ProofOfPossessionGas + GetValidatorGas` because those are its main elements.
I wasn't as worries about it because I figured we'd revise these latter.

Reply:
Actually I realized `GetParentSealBitmap` has about 12 times the history length as `BLOCKHASH` on baklava where the epoch size is 720. On mainnet it will be 24 times longer than that. Meaning that if I am still using the `BLOCKHASH` gas cost as a guide it's `20 * 12 * 24 = 6000`. Adjusted it once more.

Reply:
I'm probably missing something obvious, but why is it `12 * 24` times as expensive? Should we make the cost a function of the epoch length then, rather than hard code an assumed epoch length?